packages/core/src/components/fl-badge/fl-badge.tsx

```diff
@@ -0,0 +1,41 @@
+import { Component, Host, h, Prop, Element } from "@stencil/core";
+import { btypes, positions } from "./btypes";
+
+@Component({
+  tag: "fl-badge",
+  styleUrl: "fl-badge.scss",
+  shadow: true,
+})
+export class FlBadge {
+
+@Prop() type: btypes = btypes.PRIMARY;
+
+@Prop() pill = false;
+
+@Prop() position: positions = positions.NONE;
+
+/* stylelint-disable */
```

Comment:
Why is there a `stylelint-disable` comment?

Reply:
I kept getting a lint error that said I could only declare HTMLElement as an appropriate type but I needed to use the autogenerated HTMLBadgeElement in order to reference the self element properly.

Reply:
I just removed it thanks to your later comments.